content/docs/command-reference/status.md

```diff
@@ -156,24 +156,24 @@ that.
 
 ```dvc
 $ dvc status
-bar.dvc:
-        changed deps:
-                modified:      bar
+bar:
+	changed deps:
+        	modified:           bar
```

Comment:
As for the changes. If this is what DVC outputs, then it is correct. We'll trust you on this one to actually run each commit.